src/sinks/kafka.rs

```diff
@@ -209,9 +209,11 @@ impl Sink for KafkaSink {
 }
 
 fn healthcheck(config: KafkaSinkConfig) -> super::Healthcheck {
-    let consumer: BaseConsumer = config.to_rdkafka().unwrap().create().unwrap();
+    let client = config.to_rdkafka().unwrap();
```

Comment:
This is technically a config, not a client, but probably not a big deal.

Reply:
Yeah, probably not but doesn't hurt. I think its better to just enforce that we are doing it lazily.

Reply:
I think @lukesteensen was commenting on the variable name.

Reply:
Correct :smile:

Reply:
hahah oops yeah